backend/settings.py

```diff
@@ -218,6 +219,11 @@
         "default_link_target": "_blank",
     }
 
+# Watchman settings (https://github.com/mwarkentin/django-watchman)
+WATCHMAN_TOKENS = os.getenv('DJANGO_WATCHMAN_TOKENS', None)
+WATCHMAN_TOKEN_NAME = os.getenv('DJANGO_WATCHMAN_TOKEN_NAME', 'token')
+WATCHMAN_CHECKS = ('watchman.checks.caches', 'watchman.checks.databases')
```

Comment:
Curious question, what is it that you are caching in CAE? purpose of the caching in APP?

Reply:
Good question, I believe it's just database caching, but not sure. See `DB_CACHE_CONFIGS` in `settings.py`.

Reply:
Yeah. It is DB caching and MyLA too has it for storing the PyLTI library token info w.r.t launch. Like how you created a Browser cookie for LTI user as a JWT appending as a header in CCM. 

Now thinking back on the cache watchman status endpoint, probably that is not strictly needed since the cache is stored in DB. So a status check on DB is sufficient. But I might not know if there is any other way this app uses cache.

Reply:
Yeah, I'm not sure. I don't think there's any harm in including this, so I'm inclined to keep it as is for now.